composer.json

```diff
@@ -25,7 +25,7 @@
     },
     "require": {
         "php": "^7.1",
-        "mimmi20/wurflcache": "^1.3",
+        "roave/doctrine-simplecache": "^1.1",
```

Comment:
We should explicitly require the `psr/simple-cache` package too

Reply:
done